### PR TITLE
Kube deploy refactor

### PR DESCRIPTION
**Pull request by ce0la**
Pre-submit checklist:
- Branch
    - [ ] Tests are provided (if possible)
    - [ ] [Test report is updated](https://github.com/input-output-hk/marlowe-cardano/blob/main/marlowe/test/test-report.md) (if relevant)
    - [ ] Commit sequence broadly makes sense
    - [ ] Key commits have useful messages
    - [ ] Relevant tickets are mentioned in commit messages
    - [ ] Formatting, PNG optimization, etc. are updated
- PR
    - [ ] Self-reviewed the diff
    - [ ] Useful pull request description
        - Review required
        - [ ] Substantial changes to code, test, or documentation
        - [ ] Change made to Marlowe validator (@bwbush and @palas must be included as reviewers)
        - Review not required
        - [ ] Minor changes to non-critical code, documentation, nix derivations, configuration files, or scripts
        - [ ] Formatting, spelling, grammar, or reorganization
    - [ ] Reviewer requested

### CODE REVIEW

File: deploy/marlowe-runtime/templates/chain-indexer.yaml [hunk 1]
```diff
@@ -4,9 +4,11 @@ apiVersion: core.oam.dev/v1beta1
 kind: Application
 metadata:
   name: chain-indexer-{{ $network }}
-  namespace: marlowe-staging
+  namespace: {{ $.Values.namespace }}
   annotations:
     app.oam.dev/publishVersion: {{ $.Chart.AppVersion }}
+    meta.helm.sh/release-name: {{ $.Values.releaseName }}
```
Comment (shlevy) [inline deploy/marlowe-runtime/templates/chain-indexer.yaml:10]: Why are these needed? What do they mean?

File: deploy/marlowe-runtime/templates/marlowe-web-server.yaml [hunk 2]
```diff
@@ -35,7 +37,7 @@ spec:
       - type: https-route
         properties:
           domains:
-          - marlowe-runtime-{{ . }}-web.scdev.aws.iohkdev.io
+          - marlowe-runtime-{{ . }}-refactor-web.scdev.aws.iohkdev.io
```
Comment (shlevy) [inline deploy/marlowe-runtime/templates/marlowe-web-server.yaml:40]: Should we remove this now?

File: deploy/marlowe-runtime/values.yaml [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  - preprod
+  - preview
+  - mainnet
+
+images:
+  tag: latest
+  repo: ghcr.io
+  org: input-output-hk
+
+namespace: marlowe-staging-refactor
```
Comment (shlevy) [inline deploy/marlowe-runtime/values.yaml:11]: Back to `marlowe-staging`?

**Review by shlevy — APPROVED** [review]

File: deploy/marlowe-runtime/templates/chain-indexer.yaml (hunk 1 above)
Comment (ce0la) [reply deploy/marlowe-runtime/templates/chain-indexer.yaml:10]: The deployments were failing without the annotations being present; seems they are for KubeVela to track Helm deployments.

File: deploy/marlowe-runtime/values.yaml (hunk 3 above)
Comment (ce0la) [reply deploy/marlowe-runtime/values.yaml:11]: Yes.

File: deploy/marlowe-runtime/templates/chain-indexer.yaml (hunk 1 above)
Comment (shlevy) [reply deploy/marlowe-runtime/templates/chain-indexer.yaml:10]: Is there any documentation on them? Is this why chart changes weren't always propagating?

Comment (ce0la) [reply deploy/marlowe-runtime/templates/chain-indexer.yaml:10]: I don't think that is why chart changes weren't propagating. There are no documentations in Kubevela itself, but this SO answer captures an explanation for it: https://stackoverflow.com/a/69005327
